rust/ql/lib/codeql/rust/dataflow/internal/ModelsAsData.qll

```diff
@@ -52,6 +52,7 @@ private import codeql.rust.elements.internal.CallExprBaseImpl::Impl as CallExprB
 /**
  * Holds if in a call to the function with canonical path `path`, the value referred
  * to by `output` is a flow source of the given `kind`.
+ * The `madId` is the data extension row number.
```

Comment:
Putting "the" in front of a parameter name sounds odd to me? What about this?
```suggestion
 * Holds if in a call to the function with canonical path `path`, the value referred
 * to by `output` is a flow source of the given `kind` and `madId` is the data
 * extension row number.
```
Same suggestion for the two similar cases below.